--- ./client-participation/js/strings.js	2023-02-02 10:50:14.972066400 +0200
+++ ../polis-digifinland-tools/client-participation/js/strings.js	2022-12-01 11:38:11.468841700 +0200
@@ -22,9 +22,6 @@
   // Spanish
   es: require("./strings/es_la.js"),
 
-  // Farsi
-  fa: require("./strings/fa.js"),
-
   // French
   fr: require("./strings/fr.js"),
 
@@ -60,6 +57,13 @@
 
   // Russian
   ru: require("./strings/ru.js"),
+
+  // Finnish
+  fi: require("./strings/fi.js"),
+
+  // Swedish
+  sv: require("./strings/sv.js"),
+
 };
 
 
@@ -107,9 +111,6 @@
     else if (languageCode.match(/^es/)) {
       _.extend(strings, translations.es);
     }
-    else if (languageCode.match(/^fa/)) {
-      _.extend(strings, translations.fa);
-    }
     else if (languageCode.match(/^fr/)) {
       _.extend(strings, translations.fr);
     }
@@ -140,6 +141,17 @@
     else if (languageCode.match(/^ru/)) {
       _.extend(strings, translations.ru);
     }
+    else if (
+      languageCode.match(/^fi/) ||
+      languageCode.match(/^fi-FI/)) {
+      _.extend(strings, translations.fi);
+    }
+    else if (
+      languageCode.match(/^sv/) ||
+      languageCode.match(/^sv-SE/)) {
+      _.extend(strings, translations.sv);
+    }
+
   });
 });
 
