--- ./client-admin/src/components/landers/signout.js	2023-03-31 09:27:29.641220000 +0300
+++ ../patches/client-admin/src/components/landers/signout.js	2023-03-31 14:15:32.196733000 +0300
@@ -11,7 +11,7 @@
 @connect((state) => state.signout)
 class SignOut extends React.Component {
   componentDidMount() {
-    this.props.dispatch(doSignout('/home'))
+    this.props.dispatch(doSignout('/signin')) // DF customization, due to WAF restrictions redirect back to /signin instead of /home
   }
 
   render() {
